internal/zinc-persist/src/main/protobuf/schema.proto

```diff
@@ -0,0 +1,444 @@
+syntax = "proto3";
+
+package sbt.internal.inc;
+
+///////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////// ANALYSIS API ////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+message Hash {
+    string hash = 1;
```

Comment:
I have addressed Dale's reviews further up this PR, the decision is to keep this a string both in the Scala internal API and the internal protobuf one. Let me know if that's ok for you.